src/main/java/Factory.java

```diff
@@ -0,0 +1,72 @@
+import java.io.BufferedReader;
+import java.io.File;
+import java.io.FileReader;
+import java.io.IOException;
+
+public class Factory {
+    private int averageSalary;
+    private int minSalary = 1000_000;
+    private int maxSalary = 1;
+    private int itWorker;
+    private int supportWorker;
+    private int managementWorker;
```

Comment:
To raczej nie nie powinny być pola. Lepiej niech to będą tymczasowe zmienne w metodzie. Pozbywamy się wtedy ryzyka, że w tej klasie pozostaną dane z poprzedniego uruchomienia. Możliwe, że w tym przypadku byłoby to do ogarnięcia, ale to taka ogólna zasada, którą warto się kierować.